client/asset/btc/btc.go

```diff
@@ -5471,8 +5462,9 @@ func (btc *baseWallet) RefundBond(ctx context.Context, ver uint16, coinID, scrip
 // BondsFeeBuffer suggests how much extra may be required for the transaction
 // fees part of required bond reserves when bond rotation is enabled.
 func (btc *baseWallet) BondsFeeBuffer() uint64 {
-	// 150% of the fee buffer portion of the reserves.
-	return 15 * bondsFeeBuffer(btc.segwit, btc.feeRateLimit()) / 10
+	feeRate := btc.targetFeeRateWithFallback(1, btc.fallbackFeeRate())
+	// 110% of the fee buffer portion of the reserves.
+	return 11 * bondsFeeBuffer(btc.segwit, feeRate) / 10
```

Comment:
As a reminder, `BondsFeeBuffer` informs frontend, and gates the progress through the registration forms.  Then `ReserveBondFunds` is used just prior to posting the first bond.  As such, by making the fee rate come from a live estimate, it's possible that the live estimate will go up before they actually hit submit and it calls `ReserveBondFunds`.
For this reason, I might put a multiplier on `feeRate`, while using 110% for the entire buffer.

Thoughts?